### PR TITLE
Added Date, file, some mysqli extensions, superglobals and fixed some typos

### PR DESCRIPTION
**Pull request by Rancho2002**
I have added Date function, some important file functions and mysqli extensions and fixed some wrongly written variable name.

### CODE REVIEW

**Comment by Rancho2002** [comment]
@onecompiler-ops pls merge this pull request

File: php.md [hunk 1]
```diff
@@ -41,18 +41,23 @@ However, below are the different datatypes in PHP.
...
-define(name, value, case-insensitive)
+define(name, value, case_insensitive)
 ```
 * **name**: name of the constant
 * **value**: value of the constant
-* **case-insensitive**: Specifies whether the constant name should be case-insensitive or not. By default it is set to false.
+* **case_insensitive**: Specifies whether the constant name should be case-insensitive or not. By default it is set to false.
+
+```php
+define("NAMASTE","Hello, Greetings!");
```
Comment (onecompiler-ops) [inline php.md:58]: Change the example to globally acceptable examples

File: php.md [hunk 2]
```diff
@@ -93,7 +102,7 @@ $directions = array("East", "West", "North", "South");
 
 ### 2. Associative arrays
 ```php
-$capitals = array("Japan" => "Tokyo", "India" => "New Delhi","United Kingdom" => "London","United States" => "Washington, D.C.","China" => "Beijing");
+$capitals = array("Japan" => "Tokyo", "India" => "New Delhi","United Kingdom" => "London","United States" => "Washington, D.C.","China" => "Beijing", "Arijit" => "Student");
```
Comment (onecompiler-ops) [inline php.md:105]: revert this

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

**Comment by Rancho2002** [comment]
kindly check now and pls tell if I need to change something more @onecompiler-ops

File: php.md [hunk 3]
```diff
@@ -217,10 +226,24 @@ function functionName() { // defining a function
 functionName (parameters); //calling a function 
 ```
 
-## MySQLi Functions
-These functions allow you to access MySQL database server.
+## Date
+(PHP 4, PHP 5, PHP 7, PHP 8)
+```php
+date_default_timezone_set('Asia/Kolkata');// set default time zone( Useful while deploying to server)
```
Comment (onecompiler-ops) [inline php.md:232]: Change this to GMT/ any USA timezone

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

**Comment by Rancho2002** [comment]
please check now @onecompiler-ops

**Pull request merged**